script/Ciel/Day 12/12_01_CIEL12_3B.txt

```diff
@@ -489,7 +489,7 @@ Let's all have lunch together. I heard that the Indian pies at the café next to
 [sha:0838227d0ba55239bb3e70742d1eaf0506fc548e]{
 -- Page 127, Offset 31191.
 -- 「おっ、アーネンエルベとは通だね先輩！　あの喫茶店のマスター、どこぞのイタリア料理の達人らしいぜ！」
-"Oh, you're a connoisseur of Ahnenerbe, are you Senpai?
+"Oh, you're a connoisseur of Ahnenerbe, are you, Senpai?
```

Comment:
Hmm only one of those latter commas can stay
> Oh, you're a connoisseur of Ahnenerbe are you, Senpai?

maybe drop that `are you` as well?
> Oh, you're a connoisseur of Ahnenerbe, Senpai?

Reply:
I'm going with the latter.